src/CodeControl.jl

```diff
@@ -0,0 +1,156 @@
+export @with_stdout, @cond, @cond_with_stdout, @bind_cond, stdout_style, stdout_styles
+
+"""
+     Dict containing predefined style sheets for `div.stdout`
+"""
+_predefined_css=Dict(
+
+    :default =>"""<style>
+         div.stdout {}
+         div.stdout pre {
+                color: #000;
+                    border-radius: 1px;
+                background-color: #dfd;
+                    border: 0px solid #ddd;
+                font-size: 65%;
+          }
+           </style>""",
+
+    :nowrap =>"""<style>
+         div.stdout {}
+         div.stdout pre {
+                color: #000;
+                border-radius: 1px;
+                background-color: #dfd;
+                border: 0px solid #ddd;
+                font-size: 65%;
+                word-break: normal !important;
+                word-wrap: normal !important;
+                white-space: pre !important;
+      	        overflow: auto;
+          }
+           </style>""",
+
+    :vintage =>"""<style>
+              div.stdout {}
+              div.stdout pre {
+                color: #2f2;
+                border-radius: 10px;
+                background-color: #333;
+                border: 5px solid #aaa;
+                font-size: 65%;
+          }
+          </style>""")
+
+# Initialize with default css
+_stdout_css=_predefined_css[:default]
+
+"""
+   Set style sheet for output of stdout. Either one of
+   $(keys(_predefined_css)) or a string containing a css style sheet.
+"""
+function stdout_style(s)
```

Comment:
This function works with a side effect - it won't work well inside Pluto

Reply:
In fact it does - changing this modifies the appearance of all outputs once.
OTOH we might want to have this as a configuration option, see below.